buildpack.toml

```diff
@@ -87,6 +87,14 @@ api = "0.7"
     stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
     uri = "https://download.newrelic.com/php_agent/archive/10.1.0.313/newrelic-php5-10.1.0.313-linux.tar.gz"
     version = "10.1.0"
+  
+  
+  [[metadata.dependencies]]
+    id = "new-relic-python"
+    name = "New Relic Python Agent"
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    version = "8.1.0"
+
```

Comment:
There are two ways you can manage dependencies. You can have the buildpack install it. This would be if there's a tar/gz/archive of some sort that you want the buildpack to download and extract into the filesystem, or you can let the tools do the installation. If you let the tools do the installation then this section is not needed.

It sounds like you want pip to install the library, so my suggestion would be to remove this.

```suggestion
```